packages/block-editor/src/components/block-draggable/draggable-chip.js

```diff
@@ -49,6 +46,9 @@ export default function BlockDraggableChip( { clientIds } ) {
 							)
 						) }
 					</FlexItem>
+					<FlexItem>
+						<BlockIcon icon={ handle } />
+					</FlexItem>
```

Comment:
What's the purpose of moving this?

Reply:
So that the block icon and drag handle are in the same order on the chip as they are on the toolbar. 

It didn't feel right the other way around.

Reply:
Fair enough.

Come to think of it... why do we even show the handle icon on the drag chip? It's already pretty apparent that the user is dragging once you start, well, doing it. It seems kind of redundant to me.